newsfragments/README.md

```diff
@@ -0,0 +1,24 @@
+This directory collects "newsfragments": short files that each contain
+a snippet of ReST-formatted text that will be added to the next
+release notes. This should be a description of aspects of the change
+(if any) that are relevant to users. (This contrasts with the
+commit message and PR description, which are a description of the change as
+relevant to people working on the code itself.)
+
+ Each file should be named like `<ISSUE>.<TYPE>.rst`, where
+`<ISSUE>` is an issue number, and `<TYPE>` is one of:
+
+* `feature`
+* `bugfix`
+* `doc`
+* `misc`
+
+ So for example: `123.feature.rst`, `456.bugfix.rst`
+
+ If the PR fixes an issue, use that number here. If there is no issue,
```

Comment:
Maybe I already had this wrong in the places where I introduced this but I just noticed that there are whitespace errors in this file here

Reply:
Yikes! Thanks!